src/conda.js

```diff
@@ -51,7 +51,10 @@ async function setupConda() {
   }
 
   // Always install the latest version of showyourwork
-  exec("pip install -U showyourwork", "Install showyourwork");
+  const version = core.getInput("showyourwork-version");
```

Comment:
Doesn't look like the `showyourwork-version` option has any effect on the code.

Reply:
That's correct - I am experimenting :D The actual parameter is currently `pip-install-syw`.

Reply:
Got it. Carry on!